### ISSUE TITLE
www.britishgas.co.uk - site is not usable

### ISSUE THREAD

**Issue by miketaylr** [webcompat/pretriage-test-issues#69]
**URL**: https://www.britishgas.co.uk/apps/britishgas/components/SmartPPVendServlet/POST.servlet

**Browser / Version**: Firefox Mobile 62.0
**Operating System**: Android 7.1.2
**Tested Another Browser**: Yes

**Problem type**: Site is not usable
**Description**: page does not load
**Steps to Reproduce**:

_From [webcompat.com](https://webcompat.com/) with ❤️_

**Comment by abhiaecs** [comment]
Thanks for the report. I was able to reproduce the issue. (Firefox Focus 6.1.1)
![image](https://user-images.githubusercontent.com/10229535/45677964-12fc3b80-bb53-11e8-8528-8eb58606fabe.png)

However, when I removed two parts of the path, I got access denied error.
![image](https://user-images.githubusercontent.com/10229535/45678061-450d9d80-bb53-11e8-8d21-ac05ac21c632.png)

And finally by entering only the domain I was able to access the site.
![image](https://user-images.githubusercontent.com/10229535/45678025-30310a00-bb53-11e8-9c15-95623573eccb.png)

**Comment by abhiaecs** [comment]
I don't think it is a compatibility issue either as I was facing the same issue when trying to access through Chrome.
